lib/Transform/operators_supported.json

```diff
@@ -1704,6 +1704,10 @@
 		{
 			"name": "xten.conv2d_tensoradd_lrelu_globalaveragepool",
 			"properties": []
+		},
+		{
```

Comment:
You probably need some changes to the visualizer implementation too. Can you check a similar PR to see what is needed there?